doc/tutorial.md

```diff
@@ -0,0 +1,175 @@
+# Getting Started with Submitting Job
+
+## Download and Configure paddlectl
+
+`paddlectl` is a command-line tool that submits distributed training job to the cloud.
+
+- Step1
+
+  Download `paddlectl` and copy the binary file to the folder which under
```

Comment:
Download `paddlectl` to your system `$PATH` directory, and make it executable by running the command below

 `chmod +x paddlectl`

Stable `paddlectl` binary can be found from the [Release Page](https://github.com/PaddlePaddle/cloud/releases).

Or if you wish to try the latest one from our CI, please find the URLs from the table below for different OSs.